sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -141,6 +141,18 @@ def GetHeaders(cosmos_client_connection,
     if options.get('offerThroughput'):
         headers[http_constants.HttpHeaders.OfferThroughput] = options['offerThroughput']
 
+    if options.get('contentType'):
+        headers[http_constants.HttpHeaders.ContentType] = options['contentType']
+
+    if options.get('isQueryPlanRequest'):
+        headers[http_constants.HttpHeaders.IsQueryPlanRequest] = options['isQueryPlanRequest']
```

Comment:
nit: Can we make these strings consts, so we aren't copy and pasting them? You can ignore this comment if you want, since I know it's not your fault and the change is a bit tedious.

Reply:
there are a whole bunch of places where I wanna do these changes, maybe I will in a different PR, to keep the scope of this one low.